wolfssl/internal.h

```diff
@@ -2079,6 +2079,10 @@ typedef struct Keys {
     byte   updateResponseReq:1;   /* KeyUpdate response from peer required. */
     byte   keyUpdateRespond:1;    /* KeyUpdate is to be responded to. */
 #endif
+#ifdef WOLFSSL_RENESAS_TSIP_TLS
+    byte    tsip_client_write_MAC_secret[TSIP_TLS_HMAC_KEY_INDEX_WORDSIZE];
```

Comment:
Can you back these up one space to align with rest of the block?

Reply:
Done.